dissect/extfs/extfs.py

```diff
@@ -248,12 +247,14 @@ def link(self) -> str:
 
     @property
     def link_inode(self) -> INode:
```

Comment:
I think `link_inode` is also something we should get rid of. I think it makes more sense for symlink resolution to be pushed up the layer, so basically to `fs.get(path)`.

Reply:
Removed altogether.

As discussed, one of the conceptual problem with doing symlink resolutions inside the FS is that symlinks can cross into other filesystems. 

Further, relative symlink lookups should be  resolved using the path of the parent directory, not an inode-link.